persistence/sql/persister_oauth2.go

```diff
@@ -391,50 +393,23 @@ func (p *Persister) FlushInactiveAccessTokens(ctx context.Context, notAfter time
 
 		if i != j {
 			err = p.Connection(ctx).RawQuery(
-				fmt.Sprintf("DELETE FROM %s WHERE signature in (?)", OAuth2RequestSQL{Table: sqlTableAccess}.TableName()),
+				fmt.Sprintf("DELETE FROM %s WHERE signature in (?)", OAuth2RequestSQL{Table: table}.TableName()),
 				signatures[i:j],
 			).Exec()
+			if err != nil {
+				break
```

Comment:
```suggestion
				return sqlcon.HandleError(err)
```

Reply:
Why this change? You already have `return sqlcon.HandleError(err)` at the end of the function which handles this?

Reply:
Because it's explicit

Reply:
Then you can probably just make the last line in the function `return nil`?